appdatabase/migrations/sql/1689498471_make_wallet_accounts_positions_non_negative.up.sql

```diff
@@ -0,0 +1,39 @@
+-- All wallet account positions should be positive, startig from 0.
+-- Chat account position will be always -1.
+CREATE TABLE keypairs_accounts_tmp AS SELECT * FROM keypairs_accounts;
+
+DELETE
+FROM
+	keypairs_accounts_tmp
+WHERE
+	chat = TRUE;
+
+UPDATE
+	keypairs_accounts AS ka
+SET
+	position = kat.pos
+FROM (
+    SELECT
+		ka.name,
+		ROW_NUMBER() OVER (ORDER BY kat.position) - 1 AS pos
```

Comment:
proberbly we'll face [like issue](https://github.com/status-im/status-mobile/issues/16421) again if we use `ROW_NUMBER()` , [relate fix](https://github.com/status-im/status-go/pull/3687)